lib/packages/shared-types/opensearch/main/index.ts

```diff
@@ -1,6 +1,16 @@
 import { z } from "zod";
 
 import { SeaTool } from "../../events";
+
+export class SkippableValidationError extends Error {
+  constructor(
+    message: string,
+    public metadata?: any,
```

Comment:
I think we should type it out to just be `{ hasTitle: string; hasDescription: string }` since it seems like we're using this in one place and might benefit from more explicit typing. Just my 2 cents